### PR TITLE
docs: tiny cleanup after enzyme

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves []

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: dangerfile.ts [hunk 1]
```diff
@@ -145,7 +127,6 @@ export const useWebPs = (fileNames: string[]) => {
   const newCreatedFileNames = getCreatedFileNames(danger.git.created_files)
 
   preventUsingMoment()
-  preventUsingEnzyme()
   preventUsingTestRenderer()
   preventUsingRenderRelayTree()
```
[the remark this answers is not in the document]
Comment (pvinis) [reply dangerfile.ts:131]: oo nice. i can remove that too

[the remark this answers is not in the document]
Comment (gkartalis) [reply dangerfile.ts:131]: who will do it ? 🤔 me or you? 🥇

[the remark this answers is not in the document]
Comment (pvinis) [reply dangerfile.ts]: done

[the remark this answers is not in the document]
Comment (pvinis) [reply dangerfile.ts]: :D

[the remark this answers is not in the document]
Comment (gkartalis) [reply dangerfile.ts]: Alt Text

**Pull request merged**